src/gdcdictionary/schemas/treatment.yaml

```diff
@@ -160,6 +160,39 @@ properties:
     type: number
     minimum: 0
 
+  margin_distance:
+    $ref:
+      - "_terms.yaml#/margin_distance/common"
+    type: number
+    minimum: 0
+
+  margin_status:
+    $ref:
+      - "_terms.yaml#/margin_status/common"
+    enum:
+      - Indeterminate
+      - Involved
+      - Uninvolved
+      - Unknown
+      - Not Reported
+
+  margins_involved_site:
```

Comment:
Why the difference between 'margins' and 'margin' in the name?

Reply:
I am not sure, this property is being migrated from another node, @tlicht3 would you know?